docs/generated/contribution.ts

```diff
@@ -0,0 +1,143 @@
+const Contribution = `# Contributing
+
+Thanks for your interest in contributing to Moon Light Design System!
+
+Please take a moment to review this document **before submitting a pull request**.
+
+## Pull requests
+
+**Please ask first before starting work on any new features.**
+
+Having your pull request declined after investing a significant amount of time and effort into a new feature can be disheartening. To prevent such situations, we kindly request that contributors first create [an issue](https://github.com/coingaming/moon-light/issues) to initiate a discussion about any new features. This includes tasks like adding new components, modifying tokens, or exposing internal information.
+
+It's crucial to understand that the Moon Design System is a complex ecosystem comprising \`Figma\` files, different versions for \`React\`, \`Elixir\`, and even \`Flutter\`. Ensuring consistency and feature parity across all these platforms involves numerous stakeholders and users who rely on us to avoid breaking changes. Thus, it's vital to respect this process and seek approval **before commencing work on any new features**.
+
+## Monorepo
+
+The Moon Design repo is a monorepo using \`pnpm\` workspaces. Note that we are using \`pnpm\` **version 8**.
+
+## Documentation
+
+Currently, the React documentation (located in \`docs/\`) is a Next.js app that contains examples.
+
+It is essential that each component feature has a corresponding documentation page with code examples.
+
+## Tests
+
+You can run the test suite using the following commands:
+
+\`\`\`sh
+pnpm run doc test
+\`\`\`
+
+Please ensure that the tests are passing when submitting a pull request.
+
+If you're adding new features to Moon Design, please include tests.
+
+## Commits
+
+Commits follow the [Angular Commit Message Format](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
+
+## Branches
+
+As a developer, you will you be branching and merging from \`develop\`, our base branch.
+
+Consider \`origin/master\` to always represent the latest code deployed to production.
+
+## Accessibility
+
+All features should attempt to conform to as many items on The A11Y Project's [Web Accessibility Checklist](https://a11yproject.com/checklist) as possible. If a checkbox can't be completed, the justification should be documented for future reference.
```

Comment:
elaborate more about pa11y, @salvatorecriscioneweb could guide you

Reply:
I am not sure about what's the context of this, can you explain me? :)

Reply:
I mean in the Contributions I add that components / PRs should follow this A11Y Web Accessibility https://www.a11yproject.com/ , because is the minimum basic of rules to follow, to have the accessibility